controls/radrichtextbox/document-elements/table/api.md

```diff
@@ -0,0 +1,69 @@
+--- 
+title: Properties and Methods
+page_title: Properties and Methods
+description: Properties and Methods
+slug: radrichtextbox-features-api
+tags: table, header, row, repeat
+published: True
+position: 5
+---
+
+### Properties
+
+|__Property__|__Description__|
+|---|---|
+|__LayoutMode__|Gets or sets then table layout mode.|
+|__PreferredWidth__|Gets or sets the preferred width of the table.|
+|__FlowDirection__|Gets or sets the flow direction of the table contents.|
+|__CellPadding__| Gets or sets the cell padding.|
+|__Background__|Gets or sets the table background.|
+|__TableIndent__|Gets or sets the table indent.|
+|__CellSpacing__|Gets or sets the cell spacing.|
+
+### Methods for adding or Inserting Rows
+
+|__Method__|__Description__|
+|---|---|
+|__AddRow__|Adds copy of the last TableRow in the table.|
+|__AddRowAbove(TableRow existingRow)__|Copies a specified TableRow and adds the copy above it.|
+|__AddRowAtIndex__| Adds a TableRow at a specific index.|
+|__AddRowBelow(TableRow existingRow)__|Copies a specified Telerik.Windows.Documents.Model.TableRow and adds the copy below it.|
+|__DeleteRow(TableRow row)__| Deletes a TableRow|
+ 
+
+### RadRichTextBox exposes the following methods that regard the creation or deletion of a table:
```

Comment:
That looks strange with title formatting. Let's change it to "Inserting and Deleting a Table"?